src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -664,6 +681,37 @@ private static bool ShowInStackTrace(MethodBase mb)
             return !(mb.IsDefined(typeof(StackTraceHiddenAttribute)) || (mb.DeclaringType?.IsDefined(typeof(StackTraceHiddenAttribute)) ?? false));
         }
 
+        private static bool TryResolveStateMachineMethod(ref MethodBase mb, ref Type declaringType)
```

Comment:
>ref [](start = 76, length = 3)

Instead of reusing this variable for two different declaring types, let's just use ```out``` and mb.DeclaringType on line 689.

Reply:
changed